data/brands/amenity/cafe.json

```diff
@@ -3742,6 +3742,24 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "The Alley",
+      "id": "thealley-e1ef51",
+      "locationSet": {"include": ["001"]},
```

Comment:
Is this intended for Chinese-speaking regions or globally?

这个是打算给中文使用地区还是全球？

Reply:
可以移掉周先生的 "tw" 了

Reply:
我打算給全球使用。因為目前的鹿角巷的情形很棘手，是多個國家都有少許分店。如果各自都有語言或 entry 的話，會很冗長。所以暫時先想到全球通用。

I intended for globally since The Alley, with a few branches in multiple countries, is a tricky case. It will become redundant if we create numerous entries or languages for The Alley in each country.